apps/web/components/LatestNewsSection/LatestNewsSection.tsx

```diff
@@ -57,7 +63,13 @@ export const LatestNewsSection: React.FC<LatestNewsProps> = ({
         </GridColumn>
         <GridColumn paddingBottom={0} span="6/12" hiddenBelow="md">
           <Box display="flex" justifyContent="flexEnd" paddingBottom={2}>
-            <Link {...linkResolver(overview, [...parameters])} skipTab>
+            <Link
+              href={{
+                pathname: linkResolver(overview, parameters).href,
+                ...(!fullWidth && { query: { tag: newsTag } }),
```

Comment:
Yes that would make more sense.

We were using this to avoid having a redundant `..?tag=<organization-slug>` in the URL for the main news section (because `fullWidth == true` on the front page) but it's better we move that out of this component.